proto/teleport/lib/teleterm/v1/service.proto

```diff
@@ -471,6 +473,27 @@ message ListDatabaseUsersResponse {
   repeated string users = 1;
 }
 
+// ListResourcesFilter describes different types of filters
+// that can be applied when requesting to the "ListResources" api.
+message ListResourcesFilter {
```

Comment:
I'm still unsure about this name, technically `start_key` is not a filter. Maybe it should be called `ListResourcesParams`?